app/helpers/admin/content_publisher_routes_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Admin::ContentPublisherRoutesHelper
+  def content_publisher_home_url
+    @content_publisher_home_url ||= "#{content_publisher_base_url}/content"
```

Comment:
We don't have a url at `/content` - it doesn't look like this method is used anywhere too.

Reply:
Whoops, forgot to remove this before committing.